cea/scripts.py

```
@@ -20,12 +23,42 @@ def __init__(self, script_dict, category):
+        # Ignore if there is only one parameter
+        if len(parameters) <= 1:
+            return parameters
+
+        try:
+            scenario_in_parameters = parameters.index("general:scenario")
+        except ValueError:
+            # Ignore if scenario is not in parameters
+            return parameters
+
+        if scenario_in_parameters != 0:
+            warnings.warn(f"`general:scenario` parameter found and is not the first parameter. "
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)